### PR TITLE
Adds page menus in events+places CT's. Imp page scroll. Change mob ti…

### PR DESCRIPTION
**Pull request by davidrkupton**
…tle.

#### Fixes [insert bug/issue number]
#979

#### Changes proposed in this pull request:
* Adds secondary menus to events and place profile content types,
* Improves scrolling while navigating,
* Changes mobile menu title to "Page Sections" across all CT's with secondary menus.

#### Add @mentions of the person or team responsible for reviewing proposed changes
@jimafisk

### CODE REVIEW

File: docroot/sites/all/themes/custom/boston/templates/node/node--article.tpl.php [hunk 1]
```diff
@@ -97,7 +97,7 @@
       <?php endif; ?>
       <div class="sub-nav-trigger drawer-trigger">
         <div class="sub-nav-chevron"><?php print file_get_contents(drupal_get_path('theme', $GLOBALS['theme']) . '/dist/img/subnav-toggle.svg') ?></div>
-        Article Menu
```
Comment (jimafisk) [inline docroot/sites/all/themes/custom/boston/templates/node/node--article.tpl.php]: Have you looped in with @reillyzlab about these UI name changes?

Comment (jimafisk) [inline docroot/sites/all/themes/custom/boston/templates/node/node--article.tpl.php]: Have you looped in with @reillyzlab about these UI name changes?

Comment (davidrkupton) [reply docroot/sites/all/themes/custom/boston/templates/node/node--article.tpl.php]: I discussed with @jduffy2001 and I'll bring it up with @reillyzlab tomorrow b4 merging.

Comment (jimafisk) [reply docroot/sites/all/themes/custom/boston/templates/node/node--article.tpl.php]: Ok great, if James said it was ok, I'd say you're good to merge. Thanks @davidrkupton!

**Pull request merged**